core/src/indexer/last_synced.rs

```diff
@@ -246,7 +247,32 @@ pub fn update_progress_and_last_synced_task(
                 .await;
 
             if let Err(e) = result {
-                error!("Error updating last synced block: {:?}", e);
+                error!("Error updating db last synced block: {:?}", e);
+            }
+
+            let latest = config
```

Comment:
we already have a db command above running not better to do this in 1 execute over 2 db ones?

Reply:
Batch executing now in a simple query pipeline.